shared/src/ledger/pos/vp.rs

```diff
@@ -259,6 +262,18 @@ where
                     TotalVotingPowers::try_from_slice(&bytes[..]).ok()
                 });
                 changes.push(TotalVotingPower(Data { pre, post }));
+            } else if let Some(address) = is_validator_commission_rate_key(key)
+            {
+                let pre = self.ctx.pre().read_bytes(key)?.and_then(|bytes| {
+                    CommissionRates::try_from_slice(&bytes[..]).ok()
+                });
+                let post = self.ctx.post().read_bytes(key)?.and_then(|bytes| {
+                    CommissionRates::try_from_slice(&bytes[..]).ok()
+                });
```

Comment:
for validation, we can look-up `max_commission_rate_change` here (similarly to bond/unbond looking up the slashes here)

Reply:
Should we look this up from `pre` and `post` and ensure that it is unchanged as well?